contracts/common/errors/flashloan.rs

```diff
@@ -0,0 +1,29 @@
+use super::PSP22FlashmintError;
+use ink_prelude::string::String;
+
+#[derive(Debug, PartialEq, Eq, scale::Encode, scale::Decode)]
+#[cfg_attr(feature = "std", derive(scale_info::TypeInfo))]
+pub enum FlashBorrowerError {
+    FlashloanRejected(String),
+}
+
+#[derive(Debug, PartialEq, Eq, scale::Encode, scale::Decode)]
+#[cfg_attr(feature = "std", derive(scale_info::TypeInfo))]
+pub enum FlashLenderError {
+    Custom(String),
+}
+
+impl From<PSP22FlashmintError> for FlashLenderError {
```

Comment:
For each error that is defined in OpenBrush, we should implement conversion from all modifiers.